### PR TITLE
docs: Updated the changelog for 5.3.

### PR DESCRIPTION
**Pull request by hjafarpour**
Added some missing items to the change log.

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
 * ``CREATE STREAM`` and ``CREATE TABLE`` will now allow you to create the topic if it is missing.
   To do this, specify the ``PARTITIONS`` and optionally ``REPLICAS`` in the ``WITH`` clause.
 
+* ``CREATE STREAM AS SELECT ...`` and ``CREATE TABLE AS SELECT ...`` statements will use the source
+topic partitions and replica counts for the sink topic if the config properties are not set in ``WITH`` clause.
+In case of JOIN, the left hand side topic will be used.
+This deprecates ``ksql.sink.partitions`` and ``ksql.sink.replicas`` config properties.
```
Comment (vcrfxia) [inline docs/changelog.rst]: Should we also add this to the upgrade notes?

Comment (hjafarpour) [reply docs/changelog.rst]: Yes, thanks for reminding :) Updated that one too!

File: docs/changelog.rst [hunk 2]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
 * ``CREATE STREAM`` and ``CREATE TABLE`` will now allow you to create the topic if it is missing.
   To do this, specify the ``PARTITIONS`` and optionally ``REPLICAS`` in the ``WITH`` clause.
 
+* ``CREATE STREAM AS SELECT ...`` and ``CREATE TABLE AS SELECT ...`` statements will use the source
+topic partitions and replica counts for the sink topic if the config properties are not set in ``WITH`` clause.
```
Comment (miguno) [inline docs/changelog.rst]: Continuing the next line of the bullet item without the necessary leading whitespace will break the bullet list.

If you look at the rendered file (e.g. https://github.com/confluentinc/ksql/blob/9764f2e4e9f113e7f66c5ca8026672da1dc1f285/docs/changelog.rst), you will notice that this problem exists multiple times in the file.

File: docs/changelog.rst [hunk 3]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
...
+In case of JOIN, the left hand side topic will be used.
+This deprecates ``ksql.sink.partitions`` and ``ksql.sink.replicas`` config properties.
+
+* A new config variable, ``ksql.internal.topic.replicas``, was introduced to set the replica count for
+the internal topics created by KSQL Server. The internal topics include command topic or config topic.
```
Comment (miguno) [inline docs/changelog.rst]: Bullet list is broken because the necessary leading whitespace is missing.

File: docs/changelog.rst [hunk 4]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
...
+
+* A new KSQL testing tool was added. The tool is a command line utility that enables testing KSQL statements
+without requiring any infrastructure, like Apache Kafka and KSQL clusters.
```
Comment (miguno) [inline docs/changelog.rst]: Bullet list is broken because the necessary leading whitespace is missing.

File: docs/changelog.rst (hunk 2 above)
Comment (hjafarpour) [reply docs/changelog.rst]: Ahhh, good point! Thanks @miguno , fixed it :)

File: docs/changelog.rst [hunk 5]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
 * ``CREATE STREAM`` and ``CREATE TABLE`` will now allow you to create the topic if it is missing.
   To do this, specify the ``PARTITIONS`` and optionally ``REPLICAS`` in the ``WITH`` clause.
 
+* ``CREATE STREAM AS SELECT ...`` and ``CREATE TABLE AS SELECT ...`` statements will use the source
```
Comment (JimGalasyn) [inline docs/changelog.rst]:
```suggestion
* ``CREATE STREAM AS SELECT ...`` and ``CREATE TABLE AS SELECT ...`` statements now use the source
```

File: docs/changelog.rst [hunk 6]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
...
+  topic partitions and replica counts for the sink topic if the config properties are not set in ``WITH`` clause.
```
Comment (JimGalasyn) [inline docs/changelog.rst]:
```suggestion
  topic partitions and replica counts for the sink topic if the config properties are not set in the ``WITH`` clause.
```

File: docs/changelog.rst [hunk 7]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
...
+  In case of JOIN, the left hand side topic will be used.
```
Comment (JimGalasyn) [inline docs/changelog.rst]:
```suggestion
  In case of JOIN, the left hand side topic is used.
```

File: docs/changelog.rst [hunk 8]
```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
...
+  This deprecates ``ksql.sink.partitions`` and ``ksql.sink.replicas`` config properties.
+
+* A new config variable, ``ksql.internal.topic.replicas``, was introduced to set the replica count for
+  the internal topics created by KSQL Server. The internal topics include command topic or config topic.
```
Comment (JimGalasyn) [inline docs/changelog.rst]:
```suggestion
  the internal topics created by KSQL Server. The internal topics include the command topic and the config topic.
```

**Pull request merged**